index.html

```diff
@@ -38,197 +51,234 @@ <h1 class="hero-title">
-            </div>
         </div>
+        <div id="action">
+            <button type="button" class="project-button">See Project</button>
         </div>
-    </section>
-
-    <section class="careers-card-5">
-        <div class="careers-card">
-        <img src="Images/img-placeholder.png" alt="placeholder">
+        </div>
+    </div>
+    <div class="careers-card">
+        <div class="img-Placeholder"></div>
         <div class="careers-card-footer">
-            <h3 class="careers_title">Multi-Post Stories Gain+Glory</h3>
-            <ul class="tags">
+        <h2 id="careers_title">Multi-Post Stories Gain+Glory</h2>
+        <div class="languages-category">
+            <ul id="career-tags">
                 <li class="Lang-tag">Ruby on rails</li>
                 <li class="Lang-tag">Css</li>
                 <li class="Lang-tag">Javascript</li>
                 <li class="Lang-tag">Html</li>
             </ul> 
-            <div class="action">
-                <button class="btn-primary primary-font-style">See project</button>
-            </div>
         </div>
+        <div id="action">
+            <button type="button" class="project-button">See Project</button>
         </div>
-    </section>
-
-    <section class="careers-card-6">
-        <div class="careers-card">
-        <img src="Images/img-placeholder.png" alt="placeholder">
+        </div>
+    </div>
+    <div class="careers-card">
+        <div class="img-Placeholder"></div>
         <div class="careers-card-footer">
-            <h3 class="careers_title">Multi-Post Stories Gain+Glory</h3>
-            <ul class="tags">
+        <h2 id="careers_title">Multi-Post Stories Gain+Glory</h2>
+        <div class="languages-category">
+            <ul id="career-tags">
                 <li class="Lang-tag">Ruby on rails</li>
                 <li class="Lang-tag">Css</li>
                 <li class="Lang-tag">Javascript</li>
                 <li class="Lang-tag">Html</li>
             </ul> 
-            <div class="action">
-                <button class="btn-primary primary-font-style">See project</button>
-            </div>
+        </div>
+        <div id="action">
+            <button type="button" class="project-button">See Project</button>
         </div>
         </div>
-    </section>
     </div>
-</section>
+    </div>
+    </section>
 
 
     <!-- About me section -->
 
     
-    <section id="about_me" class="center-content">
-        <h1 class="primary-font-style primary-font-size"><b>About me</b></h1>
-        <p class="supporting_text primary-font-style">
-             I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
-        </p>
-        <button class="btn-primary primary-font-style btn-resume">Get My Resume</button>
-    </section>
+    <section class="about_me_section"><div class="about-me">
```

Comment:
- [ ] I noticed your about me section overlaps both desktop and mobile versions and looks different from the design template. Kindly fix this, please.
- [ ] Also add a background image in this section is not displaying correctly, kindly fix this, please. You are expected to stick to design as much as possible.
### Your design
![2](https://user-images.githubusercontent.com/94551484/194692804-8ca60c78-c41d-4560-9582-02a3a1023df4.png)

### Figma template
![f2](https://user-images.githubusercontent.com/94551484/194692811-9de304a9-a5a2-4f61-a05b-0a705bbee6cf.png)